datajunction-clients/python/datajunction/builder.py

```diff
@@ -126,7 +132,17 @@ def create_node(
         if existing_node_dict and "name" in existing_node_dict:
             # update
             if update_if_exists:
-                existing_node = node_cls(dj_client=self, **existing_node_dict)
+                # For Cube, DJ server returns metrics & dimensions under slightly different names
+                #   that need to be re-mapped here
+                if type_ == models.NodeType.CUBE:
+                    existing_node = Cube(
+                        dj_client=self,
+                        metrics=existing_node_dict.get("cube_node_metrics"),
+                        dimensions=existing_node_dict.get("cube_node_dimensions"),
```

Comment:
Does the `/cubes/` endpoint intentionally return a dict with fields like `cube_node_metrics` instead of `metrics`? 
If yes then this bit of ugly code is necessary and I'll write test to cover it.
If not then I can put up a PR for DJServer to rename the fields.